tests/integration/Admin/OrdersTest.php

```diff
@@ -46,7 +47,44 @@ protected function _after() {
 
 	// TODO: add test for handle_bulk_update()
 
-	// TODO: add test for maybe_stop_order_email()
+
+	/**
+	 * @see Admin\Orders::maybe_stop_order_email()
+	 *
+	 * @throws WC_Data_Exception
+	 */
+	public function test_maybe_stop_order_email() {
+
+		$orders_handler = $this->get_orders_handler();
+
+		$this->assertFalse( $orders_handler->maybe_stop_order_email( false, null ) );
```

Comment:
How about using a provider to consolidate this test method down to a single assert? See [this example](https://github.com/facebookincubator/facebook-for-woocommerce/blob/epic/instagram-checkout/tests/integration/Commerce/OrdersTest.php#L45-L71) which is similar to what could be added here.

Reply:
Yeap, sure thing. My bad I didn't refactor that after testing. Done in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1567/commits/2946c43c5f40f341694ad95e38acf953c522b70c.